File: C LANGUAGE/36.c
//36. Print a pyramid pattern with n rows.
#include<stdio.h>
int main(){
	int i,j,k;
	int n=5;
	for(i=0;i<n;i++){
		for(j=n-k;j>=0;j--){
			printf(" ");
		}
		for(k=0;k<=i;k++){
			printf("* ");
		}
		printf("\n");
	}
}

File: JAVA SCRIPT/36.js
//36. Print a pyramid pattern with n rows.
var n=5;
for(var i=0;i<n;i++){
    var row='';
    for(var k=n-i;k>0;k--){
        row=row+' ';
    }
    for(var j=0;j<i;j++){
        row=row+'* ';
    }
    console.log(row);
}